### ISSUE TITLE
allow custom email settings for Amazon

### ISSUE THREAD

**Issue by colegleason** [robamler/dontprint#35]
I would like to try having Dontprint convert with kd2pdfopt and then emailing it to Amazon and requesting a conversion to mobi. I would like to see if this allows for Kindle font size selection, highlighting, notes, etc. while still maintaining the nice reflowing.

Allowing an option to put "convert" in the email subject would do this.

**Comment by emallson** [comment]
I can say that running k2pdfopt at the command line and then passing to Amazon's convert doesn't produce nice output for ACM/IEEE-style 2-column papers. 

You might try this and see how it turns out.

**Comment by colegleason** [comment]
I tried it and I see what you mean. It would be so nice to get a good MOBI out of an ACM style paper.

**Comment by robamler** [comment]
Thanks for the suggestion and for the comment! @colegleason, do you have any particular articles in mind where you would want to use this feature? If you could point me to some articles of interest, I could test whether this feature would add a benefit before I add it to Dontprint. From a technical point of view, it's fairly easy to implement this. But I'm a bit hesitant since it would add even more clutter to the preferences UI, which is already kind of overwhelming as it is.

**Comment by colegleason** [comment]
@robamler This is an article I was reading today using Dontprint where I would have liked to highlight passages on the Kindle. http://dl.acm.org/citation.cfm?id=1639663

As @emallson said, however, converting via Amazon doesn't seem to work all that well, so this feature may not be worth it.

**Comment by robamler** [comment]
Thanks, I'll try this out on the weekend just to be sure. But since you already said that it didn't work well on your machine, I'm not optimistic. Anyway, please don't hesitate to bring up further suggestions if you have an idea to improve Dontprint.

**Comment by robamler** [comment]
I ran the article you mentioned through k2pdfopt and then sent it to my kindle with the word "Convert" in the e-mail subject. The result was very poor: Each page was interpreted as a single image and Amazon didn't recognize any text in the image. It even scaled down the images by a few pixels, which resulted in blurry images. Considering this, and the fact that the feature requested here makes the UI even more complicated, I decided not to implement this.
